### PR TITLE
VIITE-1677 VIITE-1707 master road names fixes

### PR DESCRIPTION
**Pull request by pedrosag**
Since master already has the Viite-1677 code on it then it was only a matter of adding the 1707 code.

### CODE REVIEW

**Comment by pedrosag** [comment]
All tests are green in build 127.

File: viite-UI/src/utils/LinkValues.js [hunk 1]
```diff
@@ -130,6 +130,12 @@
         SuravageRoadLink:           {value: 4, description: "SuravageRoadLink"}
     };
 
+    root.RoadNameSource = {
+        UnknownSource:              {value: -1, source: "Unknown Source"},
```
Comment (Campanha) [inline viite-UI/src/utils/LinkValues.js]: would be better to change from "source" to "description" like we use in rest of LinkValues file

File: digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala [hunk 2]
```diff
@@ -175,6 +175,20 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
           .map(midPoint)
           .getOrElse(Map("success" -> false, "reason" -> ("ID:" + id + " not found")))
     }
+
+  }
+
+  get("/roadlinks/project/prefillfromvvh") {
```
Comment (Campanha) [inline digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala]: duplicate entry point

Comment (pedrosag) [reply digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala]: Already removed it mate, thank you though.

File: viite-UI/src/utils/LinkValues.js (hunk 1 above)
Comment (pedrosag) [reply viite-UI/src/utils/LinkValues.js]: Done.

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala [hunk 3]
```diff
@@ -34,7 +34,35 @@ import scala.concurrent.duration.Duration
...
+
+object RoadNameSource {
+  val values = Set(UnknownSource, ProjectLinkSource, RoadAddressSource)
+
+  def apply (value: Long): RoadNameSource = {
+    values.find(_.value == value).getOrElse(UnknownSource)
+  }
+
+  case object UnknownSource extends RoadNameSource {
+    def value = -1
```
Comment (hestrompa) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala]: Please change this so we don't have negative values.
Add it as 99, or as 0 and the others as 1 and 2

File: viite-UI/src/utils/LinkValues.js [hunk 4]
```diff
@@ -130,6 +130,12 @@
         SuravageRoadLink:           {value: 4, description: "SuravageRoadLink"}
     };
 
+    root.RoadNameSource = {
+        UnknownSource:              {value: -1, description: "Unknown Source"},
```
Comment (hestrompa) [inline viite-UI/src/utils/LinkValues.js]: Same here, use Unknown value as 99 instead of -1

**Pull request merged**